projects/azure/base-images/test-base.yaml

```diff
@@ -61,11 +61,18 @@ items:
 
         RUN go get github.com/go-bindata/go-bindata && \
             go get golang.org/x/tools/cmd/goyacc && \
-            go get golang.org/x/tools/cmd/goimports && \
             go get github.com/golang/mock/gomock && \
             go install github.com/golang/mock/mockgen && \
-            mv $GOPATH/bin/* /usr/bin/ && \
-            rm -rf $GOPATH/* $GOPATH/.cache
+            mv $GOPATH/bin/* /usr/bin/
+
+        #pin goimports to 04b5d21e commit
```

Comment:
Can you add a comment to the discussion to why.
@kargakis we should migrate to go modules at some point, and replace all these?